exporter/loadbalancingexporter/metrics_exporter.go

```diff
@@ -120,15 +130,11 @@ func (e *metricExporterImp) ConsumeMetrics(ctx context.Context, md pmetric.Metri
 		errs = multierr.Append(errs, err)
 
 		if err == nil {
-			_ = stats.RecordWithTags(
-				ctx,
-				[]tag.Mutator{tag.Upsert(endpointTagKey, endpoints[exp]), successTrueMutator},
-				mBackendLatency.M(duration.Milliseconds()))
+			e.telemetry.LoadbalancerBackendLatency.Record(ctx, duration.Milliseconds(), metric.WithAttributeSet(exp.endpointAttr))
```

Comment:
Nit: Same as for logs, I believe recording the latency metric can be done outside of the if/else statement as it's the same call either way.